database/Seeders/eggs/voice-servers/egg-teamspeak3-server.json

```diff
@@ -53,7 +55,38 @@
             "default_value": "10011",
             "user_viewable": true,
             "user_editable": false,
-            "rules": "required|integer|between:1,65535"
+            "rules": "required|integer|between:1,65535",
+            "field_type": "text"
+        },
+        {
+            "name": "Query Protocols",
+            "description": "Comma separated list of protocols that can be used to connect to the ServerQuery | \r\nPossible values are raw, ssh and http | \r\nE.g.: raw,ssh,http",
+            "env_variable": "QUERY_PROTOCOLS_VAR",
+            "default_value": "raw",
```

Comment:
The default value in `ts3server.ini` is enabling all except HTTPS `raw,http,ssh`. I'd say disabling webinterface is not the best idea, especially when `raw` is planned to be disabled by default in the near future.

Reply:
When / where did they say `raw` will be disabled?
Last update was in 06/22, you sure they will push this soon?

I changed it to the default of the documentation, I would not remove raw in my experience a lot when not most users use this.

> Default: The default starts a raw or "classic" ServerQuery, and an encrypted ServerQuery using SSH

Reply:
In their Forum announcements when posting about new HTTP(S) webquery. Disabling by default in the near future, not removing it.

Reply:
Ah okay, but would you just remove it on default because of this?
I could imagine a bunch of people wondering.

Reply:
No, I would set the default values to match the current default, which in 'ts3server.ini` by default was everything except https.

Reply:
I checked with a new TeamSpeak, default for the ini is `raw,http,ssh`
Without the ini the default should be `raw,ssh`

So you would prefer  `raw,http,ssh` right?